src/librustc_typeck/astconv.rs

```diff
@@ -2199,10 +2198,12 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
         let trait_did = bound.def_id();
         let (assoc_ident, def_scope) =
             tcx.adjust_ident_and_get_scope(assoc_ident, trait_did, hir_ref_id);
+
+        // FIXME(ecstaticmorse): Can this use `find_by_name_and_namespace` instead?
```

Comment:
Same answer as in https://github.com/rust-lang/rust/pull/69072#discussion_r379839906.